### DISCUSSION TITLE
clarity in facial features

### DISCUSSION THREAD

**Discussion by ineow1** [hacksider/Deep-Live-Cam#728] [General]
Will there be a feature/update that allows prominent facial features that makes a person to show, for ex. if someone has a gapped tooth, will it show clearly or will it be kind of there, if someone has a lop-sided face, or even hair color, changing your hair completely.

**Comment by hacksider** [comment]
We are doing some experiments. For the tooth it might be impossible unless you use some cosmetics but we are also looking for this kind of feature (maybe soon 🤫 )

**Reply by ineow1** [reply]
Thank you Love ylu

**Reply by ineow1** [reply]
I also noticed lip color is weird too will that be apart of that

**Comment by Tomcruise3** [comment]
Love you too 😘
